### PR TITLE
[Merged by Bors] - chore: make two arguments of Finsupp.total implicit

### PR DESCRIPTION
**Pull request by PatrickMassot**
See discussion at https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Finsupp.2Etotal.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/LinearAlgebra/Finsupp.lean [hunk 1]
```diff
@@ -589,66 +589,66 @@ end LComapDomain
 
 section Total
 
-variable (α) (M) (R)
+variable (R)
 variable {α' : Type*} {M' : Type*} [AddCommMonoid M'] [Module R M'] (v : α → M) {v' : α' → M'}
 
 /-- Interprets (l : α →₀ R) as linear combination of the elements in the family (v : α → M) and
```
Comment (vihdzp) [inline Mathlib/LinearAlgebra/Finsupp.lean:595]:
```suggestion
/-- Interprets (l : α →₀ R) as a linear combination of the elements in the family (v : α → M) and
```

File: Mathlib/Algebra/Module/FinitePresentation.lean [hunk 2]
```diff
@@ -57,7 +57,7 @@ and the kernel of the presentation `Rˢ → M` is also finitely generated.
 -/
 class Module.FinitePresentation : Prop where
   out : ∃ (s : Finset M), Submodule.span R (s : Set M) = ⊤ ∧
-    (LinearMap.ker (Finsupp.total s M R Subtype.val)).FG
+    (LinearMap.ker (Finsupp.total R (Subtype.val : s → M))).FG
```
Comment (eric-wieser) [inline Mathlib/Algebra/Module/FinitePresentation.lean:60]: optional: could change to 
```suggestion
    (LinearMap.ker (Finsupp.total R ((↑) : s → M))).FG
```
here and below

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

**Comment by PatrickMassot** [comment]
bors merge

**Pull request closed without merging**